charts/trino/values.yaml

```diff
@@ -146,3 +146,19 @@ worker:
     # requests:
     #   cpu: 100m
     #   memory: 128Mi
+
+ingress:
+  enabled: false
+  className: ""
+  annotations: {}
+    # kubernetes.io/ingress.class: nginx
```

Comment:
All supported version use
```
ingressClassName: {{ .Values.ingress.className }}
```
rather than an annotation so let's remove this.

Reply:
updated